modules/internal/DefaultRectangular.chpl

```diff
@@ -167,7 +167,7 @@ module DefaultRectangular {
       this.dist = dist;
     }
 
-    proc intIdxType type {
+    proc chpl_integralIdxType type {
```

Comment:
This didn't get a deprecation because it is an internal class that shouldn't be in any user code right?

Reply:
That's right.